src/nvim/api/extmark.c

```diff
@@ -519,6 +523,8 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id, Object start, Object e
 ///                   the extmark end position (if it exists) will be shifted
 ///                   in when new text is inserted (true for right, false
 ///                   for left). Defaults to false.
+///               - no_undo_restore : Don't restore the exact position of the mark
```

Comment:
Would it be better to invert this to `undo_restore` and default to `true`?

Reply:
sure we can do that. 

I will leave this open a bit longer to tomorrow likely so now is a good time for more bikeshedding suggestions.